datus/storage/backends/vector/__init__.py

```
@@ -0,0 +1,38 @@
+    "Condition",
+    "FilterExpr",
+    "FilterCompiler",
+    "Not",
+    "Op",
+    "Or",
+    "TableSpec",
+    "VectorBackend",
+    "VectorTable",
+    "register_vector_backend",
+    "LanceBackend",
+]
```

Ruff (0.15.2)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)